firefox-ios/Client/Application/RemoteSettings/RemoteSettingsUtils.swift

```diff
@@ -30,4 +30,42 @@ class RemoteSettingsUtils {
         }
         return nil
     }
+
+    func remoteSettingsAppContext() -> RemoteSettingsContext {
```

Comment:
given the targeting requirements for this, maybe it'l be more successful as a private function of `BrowserProfile` than as a setting on this?

Reply:
That seems like a great place for this, I moved it there.